core/kazoo_number_manager/src/knm_maintenance.erl

```diff
@@ -151,6 +200,16 @@ fix_docs({ok, NumDoc}, Doc, AccountDb, NumberDb, DID) ->
             ok
     end.
 
+-spec fix_unassign_doc(ne_binary()) -> 'ok'.
+fix_unassign_doc(DID) ->
+    Routines = [{fun knm_phone_number:set_used_by/2, undefined}
```

Comment:
Tickies are so 2015 ;)

But anyway I prefer to be consistence with the current format of the file which is not used tickies at all